runtime/vm/MHInterpreter.inc

```diff
@@ -59,11 +59,7 @@ static const char * const names[] = {
 	"J9_METHOD_HANDLE_KIND_FILTER_ARGUMENTS",
 	"J9_METHOD_HANDLE_KIND_VARHANDLE_INVOKE_EXACT",
 	"J9_METHOD_HANDLE_KIND_VARHANDLE_INVOKE_GENERIC",
-#if defined(J9VM_OPT_PANAMA)
-	"J9_METHOD_HANDLE_KIND_NATIVE",
-#else /* defined(J9VM_OPT_PANAMA) */
 	"",
```

Comment:
Please remove this (blank) entry corresponding to `J9_METHOD_HANDLE_KIND_NATIVE` which has been removed.

Reply:
Fixed.